org.envirocar.app/src/org/envirocar/app/injection/BaseInjectorActivity.java

```diff
@@ -73,4 +88,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return false;
     }
 
+    public void GotoMYTRACKS(View view) {
+        navigationBottomBar.setSelectedItemId(R.id.navigation_my_tracks);
+    }
```

Comment:
not to be done here.
Please follow the proper structure.
Do this on DashboardFragment 
for eg-: BottomNavigationView bottomView= getActivity.findViewById(R.id.navigation);
then use ButterKnife @Onlick on user statistics view id 
then you can select item